app/src/test/java/com/github/se/signify/model/user/UserViewModelTest.kt

```diff
@@ -44,6 +44,26 @@ class UserViewModelTest {
     verify(userRepository).getUserName(eq(currentUserId), any(), any())
   }
 
+  @Test
+  fun updateUserNameRequestCallsRepository() {
+    val newName = "newNameTest"
+    userViewModel.updateUserName(currentUserId, newName)
+    verify(userRepository).updateUserName(eq(currentUserId), eq(newName), any(), any())
+  }
+
+  @Test
+  fun getProfilePictureUrlRequestCallsRepository() {
+    userViewModel.getProfilePictureUrl(currentUserId)
+    verify(userRepository).getProfilePictureUrl(eq(currentUserId), any(), any())
+  }
+
+  @Test
+  fun updateProfilePictureUrlRequestCallsRepository() {
+    val newUrl = "testUrl"
+    userViewModel.updateProfilePictureUrl(currentUserId, newUrl)
+    verify(userRepository).updateProfilePictureUrl(eq(currentUserId), eq(newUrl), any(), any())
+  }
+
```

Comment:
Testing the new functions before using them is a really good initiative. Well done.